assets/environment-interpreter/activate/activate

```diff
@@ -170,8 +167,8 @@ while true; do
       ;;
     -m | --mode)
       shift
-      if [ -z "${1:-}" ] || ! { [ "$1" == "run" ] || [ "$1" == "dev" ]; }; then
-        echo "Option --mode requires 'dev' or 'run' as an argument." >&2
+      if [ -z "${1:-}" ] || ! { [ "$1" == "run" ] || [ "$1" == "dev" ] || [ "$1" == "build" ] || [ "$1" == "build-sandbox" ]; }; then
```

Comment:
Suggestion: split this condition out into a separate statement so you can give it a descriptive name. This is kind of hard to parse as is.